projects/laji/src/app/shared-modules/document-viewer/identifications/identification/identification.component.html

```diff
@@ -0,0 +1,26 @@
+<div>
+  <span>
+    <i>{{identification.taxon}}</i> {{ identification.author }} 
+    <span *ngIf="!!identification.det || !!identification.detDate">
+      det. {{identification.det || ''}} {{ identification.detDate || '' }}
+    </span>
+    <span *ngIf="!!identification.linkings">
+      <a
+        [tooltip]="'showSpeciesCard' | translate"
+        [placement]="'right'"
+        container="body"
+        target="_blank" [routerLink]="['/taxon', ((identification.linkings.taxon.id || identification.linkings.taxon.qname) | toQName)] | localize" (click)="onTaxonLinkClick($event)">
+        <i class="glyphicon glyphicon-tag"></i>
+      </a>
+    </span>
+  </span>
+  <ng-container *ngIf="showFacts">
+    <div class="inset">
```

Comment:
Laji ui project has some helper classes such as ml-5 (gives 18px margin left) that could be handy here